### DISCUSSION TITLE
help installing package using brew

### DISCUSSION THREAD

**Discussion by FabSalva** [Homebrew/discussions#2455] [Getting started]
Hi, 

I am new to Homebrew, so I hope someone can help me with a possibly trivial question. 
I have installed the data analysis package 'ROOT' on my laptop via the 'brew install root' command. Once it is installed, I can see it in /usr/local/bin/root. However, when I do 'which root' on my Mac, I see that root points to an old version of ROOT that I installed ages ago. I thought that after running 'brew install root' this would point directly to the new installation? Does anyone have any experience with that? 
Thanks for any help !
Cheers,
Fab

**Comment by carlocab** [comment]
What is the output of `echo $PATH`?

**Reply by FabSalva** [reply]
Thanks for pointing me to the $PATH variable. I realised that /opt/local/bin (which is where the old version of ROOT is) was listed in the $PATH before /usr/local/bin. I have now put /usr/local/bin first (export PATH="/usr/local/bin:/opt/local/bin:$PATH") and it seems that now when I do 'which root' it points to the new version in /usr/local/bin. I hope that is the correct way to do it. 
Thanks !
Fab
